cnd/src/network/swarm.rs

```diff
@@ -284,11 +284,11 @@ async fn handle_new_match(
     let (order_hbit, order_herc20) = storage
         .db
         .do_in_transaction(|conn| {
-            use crate::storage::tables;
+            pub use crate::storage::*;
```

Comment:
How is a `pub` keyword useful in a closure's block?

Reply:
It is not :sweat_smile: